cmd/Makefile.am

```diff
@@ -7,6 +7,8 @@ dist_man_MANS =
 noinst_PROGRAMS =
 noinst_LIBRARIES =
 
+CHECK_CFLAGS = -Wall -Wextra -Wmissing-prototypes -Wstrict-prototypes \
```

Comment:
Gentoo has a policy where those are not added automatically as they tend to break good builds due to new compiler releases. I was wondering if those flags should not be in the unit tests, and instead we could resort to perhaps, just, -Wall

Reply:
Interesting, do you know if they push `-Werror` by default? Just having the warnings enabled should not cause build failures.